test/unit/org/apache/cassandra/db/memtable/MemtableSizeTestBase.java

```diff
@@ -134,6 +136,15 @@ public void testSize() throws Throwable
                     execute(writeStatement, i, j, i + j);
             }
 
+            // add test to row count in memtable
+            if(memtable instanceof TrieMemtable)
```

Comment:
Is `memtable` always a `TrieMemtable`? If so, it might be worth adding an assert before this conditional. Otherwise, maybe we should add a test that is guaranteed to cover the `TrieMemtable#rowCount` implementation.

Reply:
Right! I think I should separate add add-row test from the original one.